### PR TITLE
[iOS] Display inline code formatting

### PR DESCRIPTION
**Pull request by aringenbach**
Enable basic inline code display on iOS

### CODE REVIEW

File: platforms/ios/example/Shared/WysiwygAction+Utils.swift [hunk 1]
```diff
@@ -55,6 +55,8 @@ extension WysiwygAction: CaseIterable, Identifiable {
             return .strikeThroughButton
         case .underline:
             return .underlineButton
+        case .inlineCode:
+            return .inlineCode
```
Comment (jmartinesp) [inline platforms/ios/example/Shared/WysiwygAction+Utils.swift:59]: I think this needs to be:

```suggestion
            return .inlineCodeButton
```

It wouldn't let me build the project any other way.

**Review by jmartinesp — CHANGES_REQUESTED** [review]
Had some trouble building it as is. I added a comment with the change I had to make to build the project.

File: platforms/ios/example/Shared/WysiwygAction+Utils.swift (hunk 1 above)
Comment (aringenbach) [reply platforms/ios/example/Shared/WysiwygAction+Utils.swift]: My bad, didn't commit this as I was doing things with links in the same branch 🙈

**Review by jmartinesp — APPROVED** [review]
After the changes it seems to build and work fine. LGTM!

**Pull request merged**